src/main/kotlin/vm/VM.kt

```diff
@@ -181,10 +192,15 @@ class VM(val verb: Verb) {
                     val args = mutableListOf<Value>()
                     repeat (argCount) { args.add(0, pop()) }
                     verb.symbols[name]?.also { variableID ->
-                        val subject = variables[variableID]
-                        if (subject is VFun) {
-                            push(subject.verbInvoke(c, args))
-                        } else fail(E_TYPE, "cannot invoke non-fun as fun")
+                        // Look for variable with VFun
+                        variables[variableID]?.also { subject ->
+                            if (subject is VFun) {
+                                push(subject.verbInvoke(c, args))
+                            } else fail(E_TYPE, "cannot invoke ${subject?.type} as fun")
+                        // Look for $sys.verb
```

Comment:
This is an unrelated cool hack.  If we try to execute a lambda shorthand invoke like fooFun(args), we try to run $sys.fooFun(args).  This lets us use the $sys verbs as builtins:
```
notify(user, "Hi!")
dumpDatabase()
```
It also means we can add builtins by adding verbs to $sys.

Reply:
😎